demos/sample_azure_iot_adu/sample_azure_iot_adu.c

```diff
@@ -555,23 +545,23 @@ static AzureIoTResult_t prvEnableImageAndResetDevice()
     AzureIoTADUClientInstallResult_t xUpdateResults;
 
     /* Call into platform specific image verification */
-    LogInfo( ( "[ADU] Image validated against hash from ADU\r\n" ) );
+    LogInfo( ( "[ADU] Image validated against hash from ADU" ) );
 
     if( AzureIoTPlatform_VerifyImage(
             &xImage,
             xAzureIoTAduUpdateRequest.xUpdateManifest.pxFiles[ 0 ].pxHashes[ 0 ].pucHash,
             xAzureIoTAduUpdateRequest.xUpdateManifest.pxFiles[ 0 ].pxHashes[ 0 ].ulHashLength
             ) != eAzureIoTSuccess )
     {
-        AZLogError( ( "[ADU] File hash from ADU did not match calculated hash\r\n" ) );
+        AZLogError( ( "[ADU] File hash from ADU did not match calculated hash" ) );
```

Comment:
Do we need to use AZLogError, or can we use LogError instead? Just now I'm seeing that we are mixing Log* and AZLog* in this file.